client/asset/zec/zec.go

```diff
@@ -160,7 +160,6 @@ func NewWallet(cfg *asset.WalletConfig, logger dex.Logger, net dex.Network) (ass
 		Testnet: "18232",
 		Simnet:  "18232",
 	}
-	var w *btc.ExchangeWalletFullNode
```

Comment:
With newly merged changes, you'll need to leave this now, and just change it to `var w *btc.ExchangeWalletNoAuth`.